markdown/GameGuides/EldenRing.md

```diff
@@ -0,0 +1,98 @@
+![Elden Ring](Images\eldenring_header.png "Shot by StephenCalic"){.shadowed .autosize}
+
+## Summary
+
+Feature | Supported
+--|--
+Vanilla Photo Mode | No
+Hotsampling | No
+DSR | Yes
+Custom Aspect Ratios | Yes (with tools)
+Reshade | Yes
+Ansel | No
+Graphics API | DirectX 12
+Game version | 1.10.1
+ 
+## Tools
+
+* [Camera tools by Otis_Inf](https://patreon.com/Otis_Inf)  
+**[Features](https://opm.fransbouma.com/Cameras/eldenring.htm)**: Camera control(Also in cut scenes), FoV control, HUD toggle, timestop/gamespeed, Pillarbox/letterbox removal in custom aspect ratios (UW support), Higher LODs
+
+### Support
+* [Cheat Table](https://www.nexusmods.com/eldenring/mods/48?tab=files) Useful CT if you want to re-spawn certain bosses for shots etc.
+* [Cheat Engine Tables Guide](../GeneralGuides/cheat_engine_tables.htm)
+* [WeMod Trainer](https://www.wemod.com/cheats/elden-ring-trainers) Features Day Time +1 hour, Freeze Day Time, Freeze Enemies Position and more.
+* [Internal](https://github.com/NightFyre/ELDENRING-INTERNAL)
```

Comment:
what does this do?

Reply:
I haven't tested it since I only booted ER briefly to test something important for the guide. And after the talks with Codr, I was like gonna wait for him to test it so he can tell me if it's useful or not since he is on his play-through, and we were in no rush, that was the story. :) From the first look, it can be a good alternative to WeMod for pausing enemies. But I will remove it for now until it's fully tested to see if it's truly beneficial for shots